### DISCUSSION TITLE
Newb question: mounting on Windows

### DISCUSSION THREAD

**Discussion by daitron** [haugene/docker-transmission-openvpn#2062] [Q&A] [answered]
Hi all,
Been a long day but I managed to learn enough about docker to get this running on Windows 10.

It's all perfect except for the fact that I can't work out how to mount a folder to /data per the tips in the documentation

1. The container assumes that you mount a folder to /data¶

Technically you don't have to do this, but it is by far the most manageable way of getting the downloaded files onto your host system and Transmission will store it's state there. So if you don't mount this directory then you will loose all your torrents on image updates.

So how do I ensure I'm downloading and saving my state to my HD?

**Comment by pkishino** [comment, marked as answer]
Quick Google https://medium.com/@kale.miller96/how-to-mount-your-current-working-directory-to-your-docker-container-in-windows-74e47fa104d7
